### PR TITLE
docs: add badges to readme

### LINKED ISSUE #63: Add badges in readme

### PR DESCRIPTION
**Pull request by spaenleh**
This PR adds badges to the readme page.
Capture d’écran 2022-12-14 à 17 22 50

closes #63

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Graasp Apps Query Client
 
+[![npm (scoped)](https://img.shields.io/npm/v/@graasp/apps-query-client?logo=npm)](https://www.npmjs.com/package/@graasp/apps-query-client)
+[![GitHub package.json version](https://img.shields.io/github/package-json/v/graasp/graasp-apps-query-client?color=deepskyblue&logo=github)](https://github.com/graasp/graasp-apps-query-client/releases/latest)
+![npm](https://img.shields.io/npm/dm/@graasp/apps-query-client?color=green)
```
Comment (dialexo) [inline README.md:5]: What's the difference between this and `npm (scoped)` ? I see from the badge that it is for downloads, maybe we could write it into the alternative text?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+
+![GitHub package.json dependency version (dev dep on branch)](https://img.shields.io/github/package-json/dependency-version/graasp/graasp-apps-query-client/dev/typescript)
+![npm peer dependency version (scoped)](https://img.shields.io/npm/dependency-version/@graasp/apps-query-client/peer/react?logo=react)
+![GitHub package.json dependency version (prod)](https://img.shields.io/github/package-json/dependency-version/graasp/graasp-apps-query-client/react-query?logo=react-query)
```
Comment (dialexo) [inline README.md:9]: The alternative texts does not mention which package the version is referring to.

**Review by dialexo — CHANGES_REQUESTED** [review]
Mostly LGTM, though I think we should edit the alternative texts so that they reflect what the links are referring to

File: README.md (hunk 1 above)
Comment (spaenleh) [reply README.md:5]: Yes good point !

**Review by dialexo — APPROVED** [review]

**Pull request merged**